static/js/components/EducationForm.js

```diff
@@ -30,29 +32,29 @@ class EducationForm extends ProfileFormFields {
     setEducationDegreeInclusions:   React.PropTypes.func,
   };
 
-  openEditEducationForm = index => {
+  openEditEducationForm: Function = (index: number): void => {
     openEditEducationForm.call(this, index);
   };
 
-  openNewEducationForm = (level, index) => {
+  openNewEducationForm: Function = (level: string, index: number): void => {
     openNewEducationForm.call(this, level, index);
   };
 
-  deleteEducationEntry = () => {
+  deleteEducationEntry: Function = (): void => {
     deleteEducationEntry.call(this);
   };
 
-  renderEducationLevel(level){
+  renderEducationLevel(level: {value: string; label: string}) {
```

Comment:
yeah, for some reason you separate the key/value pairs in a object type declaration with semicolons?